lib/admin/components/UserSettings.js

```diff
@@ -255,8 +255,8 @@ export default class UserSettings extends Component<Props, State> {
       .filter(p => !p.organizationId || (this.state.organization && this.state.organization.id === p.organizationId))
     const currentProject: Project = orgProjects[this.state.currentProjectIndex]
     const projectPanel = (
-      <Panel
-        header={
+      <Panel>
+        <Panel.Heading>
           <h3>
```

Comment:
The heading text renders too big. Recommend to remove `<h3>`, or use a font size a tiny bit larger than the ambient.

Reply:
I've removed the `h3` in 5353f432defd32577a1378e0857c2946989d4bcd